Makefile

```diff
@@ -7,15 +7,19 @@ all: welcome
 	@exit 1
 
 doc:
+	export STRIP_FROM_INC_PATH_LIST=$(find . -type d -name include -printf " \"$PWD/%P\"")
```

Comment:
After grepping through the makefiles, I assume that there is a focus at the subset of include dirs dependent to a specific configuration of board, cpu, modules...
Having *all* possible includes is not a build case.